static/styles/settings.scss

```diff
@@ -834,6 +834,10 @@ input[type=checkbox].inline-block {
     margin-bottom: 5px;
 }
 
+.email {
```

Comment:
I think you likely want a more specific selector. Does the `.bots_list .email` above work?